salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
No, that's actually expected. `detectedencoding` was only introduced in Salt 2015.5.0.

I'm guessing your master is also a v2014.7.x installation right?